modules/ROOT/pages/use-api-catalog-cli.adoc

```diff
@@ -0,0 +1,50 @@
+= Using API Catalog CLI
+
+After you have installed the API Catalog CLI and configured permissions, you can use the API Catalog CLI commands to autocatalog your APIs.
+
+//Command summary start
+
+[[api-catalog-commands]]
+== API Catalog Commands
+
+API Catalog CLI includes the following commands:
+
+include::partial$api-catalog.adoc[tag=summary]
+
+//Command summary end
+
+== View Command Help
+
+Use the help option to list the commands or command options and their descriptions. The following examples show how the help option is used:
+
+----
+api-catalog --help
+api-catalog create-descriptor --help
+----
+
+NOTE: You can optionally configure the `api-catalog autocomplete` command so that you can use the tab key with a partially typed command to autocomplete it. See xref:configure-cli-autocomplete.adoc[Configure Command Autocompletion] for more details.
```

Comment:
Is it accurate to list autocompletion as part of the "View Command Help" subtopic? 
I'm wondering if this information is enough to be in its own subsection, something like: 

`== Enable Command Autocompletion

Command autocompletion enables you to use the tab key with a partially typed command to autocomplete it. 
See xref:configure-cli-autocomplete.adoc[Configure Command Autocompletion] for more details on how to configure the `api-catalog autocomplete` command.
`

Reply:
This note was accidentally left in from the Beta doc, so thanks for the catch. I'll remove it. The command already has its own subsection and is listed in the commands at the top.